easybuild/easyconfigs/t/TALON/TALON-5.0-foss-2020b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'TALON'
+version = '5.0'
+
+homepage = 'https://daler.github.io/pybedtools'
+description = """TALON is a Python package for identifying and quantifying known and
+novel genes/isoforms in long-read transcriptome data sets. TALON is technology-agnostic
+in that it works from mapped SAM files, allowing data from different sequencing platforms
+(i.e. PacBio and Oxford Nanopore) to be analyzed side by side."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/mortazavilab/%(name)s/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+patches = ['TALON-%(version)s_less_strict_deps.patch']
```

Comment:
aAdding a patch file means we'll have to maintain it too, this is easier to handle via `preinstallopts`:

```python
# change strict version requirement for pysam to minimal version requirement
preinstallopts = "sed -i 's/pysam==/pysam>=/g' requirements.txt && "